### DISCUSSION TITLE
Error when trying to run dprepro on new Dakota installation

### DISCUSSION THREAD

**Discussion by Eugene-vH-Eng** [snl-dakota/dakota#139] [Q&A]
After downloading and unpacking Dakota, and confirming that Python is associated with *.py files on my system, I get the following error when invoking dprepro from a batch file on my windows system:
dprepro.py params.in template2.dat DataFile.dat 
usage: dprepro.py [-h] [--code CHAR] [--code-block "OPEN CLOSE"]
                  [--inline "OPEN CLOSE"] [--simple-parser]
                  [--json-include FILE] [--python-include FILE] [--no-warn]
                  [--output-format FMT] [--var "var=value"] [-v]
                  include infile [outfile]
dprepro.py: error: the following arguments are required: include, infile

The line in the batch file is:  dprepro.py params.in template2.dat DataFile.dat
I have also tried: dprepro params.in template2.dat DataFile.dat

I am using Dakota 6.19 and have Python 3.11.9 installed on my system.
Any suggestions on how to solve this issue will be greatly appreciated.

**Comment by Eugene-vH-Eng** [comment]
Additional information:  The operating system is Windows Server 2019 standard; and I do not have administrative privileges.

**Comment by jadamstephens** [comment]
The problem (probably) is that while you may have associated .py files with Python, some additional configuration is needed to make Windows pass along command line arguments to it. In my experience, it's not easy to do this correctly. I usually give up and just run dprepro by calling it Python directly: `python3 dprepro.py params.in template2.dat DataFile.dat`.

**Reply by Eugene-vH-Eng** [reply]
Thank you for prompt and helpful response.
That worked!
In the words of Dr. Watson: Everything is commonplace once explained.
